ddtrace/__init__.py

```diff
@@ -7,13 +7,7 @@
 
 ModuleWatchdog.install()
 
-# Acquire a reference to the threading module. Some parts of the library (e.g.
-# the profiler) might be enabled programmatically and therefore might end up
-# getting a reference to the tracee's threading module. By storing a reference
-# to the threading module used by ddtrace here, we make it easy for those parts
-# to get a reference to the right threading module.
-import threading as _threading
-
+import ddtrace.internal._unpatched  # noqa
```

Comment:
pedantic:

we don't need to do the `_` prefix on the module name since `internal` is already a private namespace (explicitly not covered as part of the public API).

however you are correct that we don't want to import this as not `_` prefixed here.

we could do: `from ddtrace.internal import unpatched as _unpatched` if we don't want to do the `_` prefix for internal usage.

again, pedantic, I truly don't care if we keep it named this way or not, just sharing a different perspective.

Reply:
I already did that for `ddtrace/internal/_exceptions.py` so I was following the same pattern. 
Main advantage is that it's clear for everyone it's private.

Reply:
🤷🏻 I think "internal" is clear, but like I said. I truly don't care, I'll resolve. Your reasoning and the consistency is good enough.